advocacy_docs/security/advisories/cve202341114.mdx

```diff
@@ -1,19 +1,19 @@
 ---
-title: EDB Postgres Advanced Server (EPAS) authenticated users may fetch any URL
-navTitle: CVE-2023-XXXXX-6
+title: CVE-2023-41114 - EDB Postgres Advanced Server (EPAS) authenticated users may fetch any URL
+navTitle: CVE-2023-41114
 ---
 
 First Published: 2023/08/21
 
-Last Updated: 2023/08/21
+Last Updated: 2023/08/28
```

Comment:
do we want to mention what the update was about?

Reply:
In the change history at the bottom.